app/subscriber/src/components/date-filter/DateFilter.tsx

```diff
@@ -16,66 +15,83 @@ export interface IDateFilterProps {
 /** Custom date filter for the subscriber home page. Control the calendar state with custom button, custom styling also applied. Also allows user to navigate a day at a time via arrow buttons. */
 export const DateFilter: React.FC<IDateFilterProps> = ({ loaded, filter, storeFilter }) => {
   /** control state of open calendar from outside components. i.e custom calendar button */
-  const [
-    {
-      mediaType: { filter: storedFilter },
-    },
-  ] = useContent();
   const [open, setOpen] = React.useState(false);
 
   // /** close calendar after a date has been selected, and fetch related content */
   React.useEffect(() => {
     setOpen(false);
   }, [filter.startDate, setOpen]);
 
-  /** update state variable when date changes, can be controlled via date picker or arrows */
+  // set initial DateFilter date
   React.useEffect(() => {
     if (!filter.startDate) {
       storeFilter({
         ...filter,
         startDate: moment().startOf('day').toISOString(),
         endDate: moment().endOf('day').toISOString(),
       });
-      return;
-    } else if (storedFilter.startDate !== filter.startDate) {
-      storeFilter({
-        ...filter,
-        startDate: moment(filter.startDate).startOf('day').toISOString(),
-        endDate: moment(filter.startDate).endOf('day').toISOString(),
-      });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [filter]);
```

Comment:
This will fire every time the filter changes.  Which would blow away a selected endDate if they had specified one.  Is this desirable?

Reply:
It's a bit confusing but the concept of the DateFilter means we are only filtering for one day, or one 24-hr period. So the end date will always be 23:59:59 hours ahead of the startDate whenever using a DateFilter anywhere.

Also this only does anything on initial setup, that's the only time `if (!filter.startDate)` would be true. After first load, there will always be a startDate. This is the logic we had before I rewrote it in my last PR. I changed it back since I thought it was actually a bit safer/clearer since the confusion yesterday.